src/pages/base/manage-entity.page.ts

```diff
@@ -0,0 +1,60 @@
+import { Page, Response } from 'playwright';
+import BrowserService from '../../utils/browser.service';
+import Configs from '../../utils/configs';
+import CiviApiService from '../../utils/cv-api.service';
+
+/**
+ * Manage Entity Page
+ */
+export abstract class ManageEntity {
+  caseTypeCategory = 'cases';
+
+  /**
+   * @param {BrowserService} browser browser object
+   */
+  constructor (public browser: BrowserService) {
+    this.browser = browser;
+  }
+
+  /**
+   * @param {Page} page page object
+   * @returns {Promise<Response|null>} promise
+   */
+  async navigate (page: Page): Promise<Response|null> {
+    return await page.goto(Configs.getSiteConfig().url + this.getPageUrl());
+  }
+
+  /**
+   * @param {Page} page page object
+   * @returns {Promise<any>} promise
+   */
+  async waitForPageLoad (page: Page): Promise<any> {
+    return await page.waitForSelector('.civicase__case-filter-panel__button');
```

Comment:
@deb1990 Please move all the selectors within page classes to variables and then reuse the variables inside the tests. To make it a bit more accessible we can create array and pop all the elements inside the array so we can refer to elements by "PageName.selectors.ElementName"

This would be useful when tests are using elements from more than one page in E2E scenarios.